contracts/access/rbac/MinterRole.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.24;
+
+import "./Roles.sol";
+
+contract MinterRole {
+  using Roles for Roles.Role;
+
+  Roles.Role private minters;
+
+  constructor(address[] _minters) {
+    minters.addMany(_minters);
+  }
+
+  function addMinter(address _account) internal {
+    minters.add(_account);
+  }
+
+  function removeMinter(address _account) internal {
+    minters.remove(_account);
+  }
+
+  function transferMinterRole(address _account) public {
```

Comment:
this function is public, which is probably fine, but should be noted that it allows the existing minters to change who has the permission

Reply:
That's intentional - we've discussed this with @frangio and see no reason why people wouldn't be able to transfer their roles.

Also, this allows to easily setup situations like this:
```
const token = await MintableToken.new([initialMinter]);
const crowdsale = await Crowdsale.new(token.address);
await token.transferMinterRole(crowdsale.address, { from: initialMinter });
```